fine_tune.py

```diff
@@ -393,7 +393,7 @@ def fn_recursive_set_mem_eff(module: torch.nn.Module):
             current_loss = loss.detach().item()  # 平均なのでbatch sizeは関係ないはず
             if args.logging_dir is not None:
                 logs = {"loss": current_loss, "lr": float(lr_scheduler.get_last_lr()[0])}
-                if args.optimizer_type.lower().startswith("DAdapt".lower()):  # tracking d*lr value
+                if args.optimizer_type.lower().startswith("DAdapt".lower()) or args.optimizer_type.lower() == "Prodigy":  # tracking d*lr value
```

Comment:
is this `args.optimizer_type.lower() == "Prodigy"

supposed to be `args.optimizer_type.lower() == "prodigy"` ?